integration-tests/test/fee-payment.spec.ts

```diff
@@ -36,17 +41,28 @@ describe('Fee Payment Integration Tests', async () => {
   it('Paying a nonzero but acceptable gasPrice fee', async () => {
     const amount = utils.parseEther('0.5')
     const balanceBefore = await env.l2Wallet.getBalance()
+    const feeVaultBalanceBefore = await env.l2Wallet.provider.getBalance(
+      predeploys.OVM_SequencerFeeVault
+    )
     expect(balanceBefore.gt(amount))
 
     const tx = await env.ovmEth.transfer(other, amount)
     const receipt = await tx.wait()
     expect(receipt.status).to.eq(1)
 
     const balanceAfter = await env.l2Wallet.getBalance()
+    const feeVaultBalanceAfter = await env.l2Wallet.provider.getBalance(
+      predeploys.OVM_SequencerFeeVault
```

Comment:
I feel like `OVM_SequencerFeeVault` is a bit redundant with the words fee and vault. Funds could still come from elsewhere to be held in there

Reply:
I'm ok with leaving it, just a nit